docs/send-data/hosted-collectors/amazon-aws/aws-security-data-lake-source.md

```diff
@@ -0,0 +1,90 @@
+6. In **Log and event sources**, you see three options again:
+     * **CloudTrail**: View user activity and API usage in AWS services.
+     * **VPC flow logs**: View details about IP traffic to and from network interfaces in your VPC.
+     * **Route S3**: For example, you select Cloudtrail to get all the user activity and API usage in AWS services.<br/> ![cloudtrail.png](/img/send-data/cloudtrail.png)
+7. Access data.
+     * **S3** or **Lakeformation** is the storage option you can choose for how your subscribers will access the data.
+
+:::note
+Subscription URL will be added to the subscriber in the later step and is kept blank deliberately.
+:::
+
+
+### Step 3. Create AWS Security Data Lake Source using Sumo Logic Environment
+
+When you create an AWS Security Data Lake source, you add it to a Hosted Collector. Before creating the source, identify the Hosted Collector you want to use from the existing ones or create a new Hosted Collector, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+
+To create an AWS Security Data Lake source, follow the instructions below:
+1. In the Sumo Logic environment, go to **Manage Data** > **Collection** > **Collection**.
+2. On the **Collectors page, click **Add Source** next to a Hosted Collector.
+3. Select an existing Hosted Collector or the one you have just created. <br/><img src={useBaseUrl('img/send-data/aws-security-lake.png')} alt="aws moose" />
+4. Select **AWS Security Lake**.
+5. Enter a name for the new **Source**. A **Description** is optional.
+6. In the **Source Category**, enter any string to tag the output collected from this distinct source. (Category metadata is stored in a searchable field called **_sourceCategory**).
+7. In **Fields**. Click the **+Add Field** link to add custom log metadata fields.
+8. Enter the required fields that you want to associate, each field needs a name (key) and value.
+  * ![green check circle.png](/img/reuse/green-check-circle.png)  A green circle with a checkmark shows up when a field exists and is enabled in the Fields table schema.
+  * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point shows up when the field doesn't exist or is disabled in the **Fields table schema**.
+
+
+:::important
+In this case, an option to automatically add or enable the nonexistent fields to the **Fields table schema** is provided. If a field is sent to Sumo logic that does not exist in the **Fields table schema** or is disabled, it will be ignored and known as dropped field.
+:::
+
+9. In **Role ARN**: Copy and paste the AWS Role ARN from the Data Lake console.
+10. Click **Save**. A pop-up will appear with the subscription URL.
+
+
+:::note
+Remember to copy and secure the subscription URL. If you closed the window and could not copy the URL, don't worry. Click on the **Edit** button under the list of sources and copy the URL.
+:::
+
```

Comment:
```suggestion
:::important
In this case, an option to automatically add or enable the nonexistent fields to the **Fields table schema** is provided. If a field is sent to Sumo logic that does not exist in the **Fields table schema** or is disabled, it will be ignored and known as dropped field.
:::
9. In **Role ARN**: Copy and paste the AWS Role ARN from the Data Lake console.
10. Click **Save**. A pop-up will appear with the subscription URL.
:::note
Remember to copy and secure the subscription URL. If you closed the window and could not copy the URL, don't worry. Click on the **Edit** button under the list of sources and copy the URL.
:::

```